components/index.scss

```diff
@@ -32,6 +32,7 @@
 @import "./button/css";
 @import "./button-group/css";
 @import "./calendar/css";
+@import "./card-list/css";
 @import "./code/css";
 @import "./code-block/css";
 @import "./color-sample/css";
```

Comment:
Deze CSS component moet even gerebased worden op `main`, en moet dezelfde directory structuur hebben als de nieuwe componenten, en er moet ook een `package.json` komen. Als idee package er is, dan moeten storybook-react en component-library-react ook een dependency krijgen op `@utrecht/card-list-css`.